src/job/jobValidator.ts

```diff
@@ -27,7 +30,7 @@ export class JobValidator implements IJobValidator {
     }
 
     async throwIfBranchNotConfigured(job: IJob): Promise<void> {
-        let response = await this._fileSystemService.downloadYaml(`https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/${job.payload.repoName}.yaml`);
+        let response = await this._fileSystemService.downloadYaml(`${this._config.get<any>('publishBranchesUrl')[this._config.get<string>('env')]}/${job.payload.repoName}.yaml`);
```

Comment:
(Nit) Should `${this._config.get<any>('publishBranchesUrl')[this._config.get<string>('env')]}` be split off into its own variable for readability?